src/allotropy/allotrope/models/adm/electrophoresis/benchling/_2024/_06/electrophoresis.py

```diff
@@ -1,6 +1,6 @@
 # generated by datamodel-codegen:
 #   filename:  electrophoresis.schema.json
-#   timestamp: 2024-07-18T20:13:51+00:00
```

Comment:
NOTE to reviewer: this is just some moving around of classes for some reason, plus using `TIntegerValue`